netpbm/install.yaml

```diff
@@ -0,0 +1,2 @@
+- type: system
+  name: libnetpbm10-dev
```

Comment:
What is netpbm for?

Reply:
It is used by the some navigation packages. We don't use these packages, but they are in the fork of the nav stack.

Reply:
OK